pkg/dumpformat/parquetfile/type_converter.go

```
@@ -184,119 +189,121 @@ func getBoolDataSetter(val bool, d *types.Datum) error {
 				}
-				val = rebased
+				if logicalType.TimeUnit() == schema.TimeUnitMillis {
+					val = rebased / 1000
+				} else {
+					val = rebased
+				}
 			}
-			t := arrow.Timestamp(val).ToTime(arrow.Microsecond)
-			if converted.IsAdjustedToUTC {
-				t = t.In(loc)
+			unit := arrow.Microsecond
```

golangci-lint (2.12.2)

[error] ineffectual assignment to unit

(ineffassign)